Proofs/SHA512/SHA512Sym.lean

```diff
@@ -47,6 +47,7 @@ abbrev ktbl_addr : BitVec 64 := 0x1b4300#64
 -- set_option profiler true in
 -- set_option profiler.threshold 1 in
 -- set_option pp.deepTerms false in
+set_option maxHeartbeats 9999999 in
```

Comment:
I'm hoping this is stopgap, right? Adding a fuel arg. and treating literals as atomic should help?

If so, please add a `FIXME` or something there to indicate it'd go away soon.

Reply:
That's right. In the final implementation I will upstream (https://github.com/leanprover/LNSym/pull/113/files#diff-51d27053b63053ca91adbfef85c82a2d7fe480198ef4b1d2cbdda5934e393de0R50), we won't have a `maxHeartbeats`.